### DISCUSSION TITLE
Long press only if still

### DISCUSSION THREAD

**Discussion by zRelux** [pmndrs/use-gesture#259] [Q&A]
Now I have some long press detection which works fine, but I would want it only to work if the user does the first touch but does not drag the div for a certain amount of time. like they have to press it but not move it.
E.g. code:
```js
      if (first) { 
        timeout.current = setTimeout(() => {console.log('long press')}, 500);
      }

     // My idea was if (last || (first && distance > 0)) but doesn't seems to work
      if (last)) {
        clearTimeout(timeout.current);
      }
```

**Comment by dbismut** [comment]
Didn't we already discuss this?
